adafruit_rgb_display/rgb.py

```diff
@@ -1,35 +1,43 @@
-from micropython import const
+""" Base class for all RGB Display devices """
 import time
+from micropython import const
 try:
     import struct
 except ImportError:
     import ustruct as struct
 
 import adafruit_bus_device.spi_device as spi_device
 
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_RGB_Display.git"
 
 # This is the size of the buffer to be used for fill operations, in 16-bit
 # units. We use 256, which is 512 bytes — size of the DMA buffer on SAMD21.
 _BUFFER_SIZE = const(256)
 
 
 def color565(r, g, b):
+    """Format color code for device"""
     return (r & 0xf8) << 8 | (g & 0xfc) << 3 | b >> 3
 
 
 class DummyPin:
     """A fake gpio pin for when you want to skip pins."""
     def init(self, *args, **kwargs):
+        """Dummy Pin init"""
         pass
 
     def low(self):
+        """Dummy low Pin method"""
         pass
 
     def high(self):
+        """Dummy high Pin method"""
         pass
```

Comment:
This whole class needs to be rewritten for CircuitPython, as it has a completely different gpio pin API.

Reply:
I am inclined to remove dummy pin for now.

Reply:
We probably should move it to simpleio, but that would be a separate pull request. Please ignore this comment for now.

Reply:
ok